recipe/meta.yaml

```diff
@@ -47,7 +47,9 @@ requirements:
     # See libevent variants' definition in `recipe/conda_build_config.yaml`
     - libevent
     - libprotobuf
-    - libmongocxx
+    # TODO: Fix builds for missing symbols
+    # See: https://github.com/man-group/ArcticDB/pull/1050
+    - libmongocxx <3.9
```

Comment:
Same remark as for #112 , see https://github.com/conda-forge/arcticdb-feedstock/pull/112#discussion_r1403437352